styles.css

```diff
@@ -1,14 +1,68 @@
 * {
-    padding-left: 10px;
+    margin: 0;
+    padding: 0;
+    border: 0;
+    outline: 0;
+}
+
+*, *::before, *::after {
+    box-sizing: border-box;
 }

```

Comment:
This is called "Reset CSS", it removes all the default spacings between blocks and gives you all control. Try to always use it or something similar.